googleapiclient/errors.py

```diff
@@ -107,6 +107,10 @@ class InvalidNotificationError(Error):
   """The channel Notification is invalid."""
   pass
 
+class InvalidMessageError(Error):
```

Comment:
On the second thought, a name like ParameterRequiredError looks more appropriate. What do you think @volnt ?